src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -327,6 +327,7 @@ public class HttpProtocolTests : HttpClientTestBase
         }
 
         [Theory]
+        [ActiveIssue(25880)]
```

Comment:
25880 suggests that these failures only occur with WinHttpHandler. True?  If so, can it be disabled in the test just for WinHttpHandler?

Reply:
Will do.